src/util/ses.ts

```diff
@@ -378,32 +386,28 @@ export function sendNFTBookPendingClaimEmail({
           Charset: 'UTF-8',
           Data: getNFTTwoContentWithMessageAndButtonTemplate({
             title1: titleZh,
-            content1: `<p>親愛的讀者：</p>
-            <p>感謝支持並購買<a href="${nftPageURLZh}">《${bookName}》</a>。</p>
-            <p>請根據網頁指示領取你的電子書，亦可按以下連結前往該頁面。完成步驟後，即可領取電子書。</p>
-            <p>若你購買的電子書需要作者親自簽發，請在完成步驟後，耐心等待作者發貨。
+            content1: `<p>親愛的${receiverDisplayName || '讀者'}：</p>
+            <p>感謝購買<a href="${nftPageURLZh}">《${bookName}》</a>。</p>
+            <p>請<a href="${claimPageURLZh}">登入 Liker Land 書架</a>閱讀你的電子書。
+            若你未註冊 Liker Land 帳號，請點擊<a href="${claimPageURLZh}">這裡</a>註冊。</p>
+            <p>若你購買的電子書需要作者親自簽發，請先點擊<a href="${claimPageURLZh}">這裡</a>登入 Liker Land，然後耐心等待作者的發貨通知。
```

Comment:
```suggestion
            <p>若你購買的電子書需要作者親自簽發，請先點擊<a href="${claimPageURLZh}">這裡</a>登入 Liker Land 登記，然後耐心等待作者的發貨通知。
```

Since reader still need to claim the book for author to send, I suggest some kind of action should be hinted, otherwise registered user will misunderstand no action has to be taken

Reply:
I agree, let's wait for Edmond and come to a final decision